sdks/python/apache_beam/runners/interactive/messaging/interactive_environment_inspector.py

```diff
@@ -36,16 +36,17 @@ class InteractiveEnvironmentInspector(object):
   list_inspectables first then communicates back to the kernel and get_val for
   usage on the kernel side.
   """
-  def __init__(self):
+  def __init__(self, ignore_synthetic=True):
```

Comment:
Same question, why do we need this boolean?

Reply:
To produce two different views of PCollections: one for the end user, the other for internal usage.

We can also do it in another way: instead of creating 2 inspectors, create 2 inner fields holding the 2 views. Let me know what you think.

Reply:
In that case, this way may be best because the change is simpler than keeping two dicts holding the two views.